forgerock-auth/src/main/java/org/forgerock/android/auth/SSOToken.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SSOToken(override val value: String,
```

Comment:
cant remember , do we need a failure URL ?

Reply:
We don't return failure when success Authenticated.